### DISCUSSION TITLE
[Solved] Troobleshooting with insights

### DISCUSSION THREAD

**Discussion by nicolas-goyon** [community/community#23373] [New to GitHub]
<p>I have a project where i have some friends with me, the thing is to have access to multiple levels of access, i had to create an organisation. I don’t know if its correlated but when i want to see on the “Insights” section to see the comits, i cannot access to network.<br>
I only have access to Dependecy graph and Fork… Anyone can help me ?</p>
<p><img src="https://user-images.githubusercontent.com/38950115/181076613-f9e989d3-ca99-4037-ab7c-a07344dc1b61.png" alt="image" data-base62-sha1="dQYVwoUc0gtVafWaBWiy09fx6K" width="379" height="480"></p>
<p>By the way, i just saw, i can’t create a new branch.</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="nicolas-goyon" data-post="1" data-topic="190792">
<div class="title">
<div class="quote-controls"></div>
 nicolas-goyon:</div>
<blockquote>
<p>By the way, i just saw, i can’t create a new branch.</p>
</blockquote>
</aside>
<p>That means you don’t have full privileges for the repository or within the org, so there’s something wrong. You might need to check the org settings and where your account fits in it.</p>
<p>I don’t know if this is the reason why you’re not seeing the <strong>Commits</strong> menu entry, but it should be there. I also belong to an org, but that menu is visible under the repository <strong>Insigths</strong>.</p>

**Comment by nicolas-goyon** [comment]
<p>The thing is i created the org and the repo, so i guess i have full privileges.</p>
<p>Privileges on the repo :<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/0/b021c10098a42e246a979635054ccb45f35db07f.png" data-download-href="/uploads/short-url/p88lPY49CnZvFjZG1a2d7gCh29N.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/38950115/181076615-981e7df5-c830-461a-b8eb-77ac6ad4547c.png" alt="image" data-base62-sha1="p88lPY49CnZvFjZG1a2d7gCh29N" width="690" height="52" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/0/b021c10098a42e246a979635054ccb45f35db07f_2_690x52.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/0/b021c10098a42e246a979635054ccb45f35db07f_2_1035x78.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/0/b021c10098a42e246a979635054ccb45f35db07f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/0/b021c10098a42e246a979635054ccb45f35db07f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1200×91 5.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nicolas-goyon** [comment]
<p>privileges on the org :<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/4/e4e006e88c3bf40874be44f41dabe63cc99a495e.png" data-download-href="/uploads/short-url/wEIQ0Bgyt5EC18JdLGaW2xiyACG.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/38950115/181076619-4c095ec8-3cb7-4195-87d1-65fb542704ed.png" alt="image" data-base62-sha1="wEIQ0Bgyt5EC18JdLGaW2xiyACG" width="690" height="53" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e4e006e88c3bf40874be44f41dabe63cc99a495e_2_690x53.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e4e006e88c3bf40874be44f41dabe63cc99a495e_2_1035x79.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/4/e4e006e88c3bf40874be44f41dabe63cc99a495e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/4/e4e006e88c3bf40874be44f41dabe63cc99a495e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1213×94 7.13 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I just solved a problem, now i can create a branch, but the main problem is still there.</p>

**Comment by nicolas-goyon** [comment, marked as answer]
<p>Problem solved :<br>
The repository must be in public or i have to upgrade git to github teams</p>

**Comment by tajmone** [comment]
<aside class="quote no-group" data-username="nicolas-goyon" data-post="5" data-topic="190792">
<div class="title">
<div class="quote-controls"></div>
 nicolas-goyon:</div>
<blockquote>
<p>The repository must be in public or i have to upgrade git to github teams</p>
</blockquote>
</aside>
<p>You mean that the “Commits” submenu in “Insights” is not visible at all, even to the repository maintainers?</p>
